src/fib/fiber.mli

```diff
@@ -17,20 +17,27 @@ type cancel_callback = Exn_bt.t -> unit
 (** Do not rely on this, it is internal implementation details. *)
 module Private_ : sig
   type 'a state
+  type pfiber
```

Comment:
I'm not sure it's worth exposing the internal details just to enable unboxing of `any` below. I think it would likely be fine to keep it boxed.

Reply:
I'd rather keep things as lightweight as possible. This is an opaque type and in `Private_` anyway :)